cmake/scripts/get_latest_tag.sh

```diff
@@ -4,6 +4,11 @@
 # current HEAD
 #
 
+if ! git --version | grep -q "git version 2"; then
+  # This script requires version 2 or newer
+  exit 1
```

Comment:
on a second though, i guess it's fine to simply return `1` as you do here so that we do not set `tag` in CMake script.